packages/mermaid/src/diagrams/flowchart/flowDb.js

```diff
@@ -416,7 +416,7 @@ const setupToolTips = function (element) {
       tooltipElem
         .text(el.attr('title'))
         .style('left', window.scrollX + rect.left + (rect.right - rect.left) / 2 + 'px')
-        .style('top', window.scrollY + rect.top - 14 + document.body.scrollTop + 'px');
+        .style('top', window.scrollY + rect.top + (rect.bottom - rect.top) + 'px');
```

Comment:
If the bracket is opened, it simplifies to this. 
Any reason we can't use it?

```suggestion
        .style('top', window.scrollY + rect.bottom  + 'px');
```